config/plugins.yaml

```diff
@@ -345,9 +345,9 @@ config_updater:
     label_sync/labels.yaml:
       name: label-config
       namespace: test-pods
-    prow/config.yaml:
+    config/config.yaml:
```

Comment:
I suspect we'll want the file to exist in both locations for at least one commit.